snooty.toml

```diff
@@ -129,5 +129,5 @@ value = """\
 targets = ["atlas-cli-admin-api.txt"]
```

Comment:
we should extend this to include all files in /command that have filenames that start with atlas-api

see https://code.visualstudio.com/docs/editor/glob-patterns